docs/elasticsearch.rst

```diff
@@ -68,6 +68,16 @@ This can just be a random alphanumeric string.  This will trigger a preindex
 as outlined in the `Indexes` section.
 
 
+Updating indexes in a production environment
+''''''''''''''''''''''''''''''''''''''''''''
+Updates in a production environment should be done in two steps, so to not show incomplete data.
+
+1. Setup a release of your branch using cchq <env> setup_limited_release:keep_days=n_days
+2. In your release directory, kick off a index using ``./mange.py ptop_preindex``
+3. Verify that the reindex has completed successfully
```

Comment:
How do you verify this? Logging into prod and checking the data (in a shell or cloudant or ES head or something depending on what you're doing)?

Reply:
https://github.com/dimagi/commcare-hq/pull/21190/commits/5a814f2ca3cdae40fe276f861120acb7bfc291bc